sumOfOther.js

```diff
@@ -0,0 +1,20 @@
+function sumOfOther(arr) {
```

Comment:
This function works, but you may have better solution.
Please recall big O notation ( now the complexity of your function is O(n^2) ). It would be great if you could offer O(n) solution :)

Mark for this function is 10, because you didn't have any restrictions about complexity.